modules/gapi/src/api/s11n.cpp

```diff
@@ -94,6 +94,9 @@ cv::GRunArgsP cv::gapi::bind(cv::GRunArgs &results)
         case cv::GRunArg::index_of<cv::RMat>() :
             outputs.emplace_back((cv::RMat*)(&(cv::util::get<cv::RMat>(res_obj))));
             break;
+        case cv::GRunArg::index_of<cv::MediaFrame>() :
+            outputs.emplace_back((cv::MediaFrame*)(&(cv::util::get<cv::MediaFrame>(res_obj))));
```

Comment:
I believe `reinterpret_cast`

Reply:
I don't get why this C-style casting is necessary at all here. Please try to remove it or replace with some correct `_cast` operator.

Reply:
Done